### ISSUE TITLE
How do I set up the evaluation config file for evaluation for a yolov6lite-m model?

### ISSUE THREAD

**Issue by jethrolow** [meituan/YOLOv6#887]
### Before Asking

- [X] I have read the [README](https://github.com/meituan/YOLOv6/blob/main/README.md) carefully. 我已经仔细阅读了README上的操作指引。

- [X] I want to train my custom dataset, and I have read the [tutorials for training your custom data](https://github.com/meituan/YOLOv6/blob/main/docs/Train_custom_data.md) carefully and organize my dataset correctly; (FYI: We recommand you to apply the config files of xx_finetune.py.) 我想训练自定义数据集，我已经仔细阅读了训练自定义数据的教程，以及按照正确的目录结构存放数据集。（FYI: 我们推荐使用xx_finetune.py等配置文件训练自定义数据集。）

- [X] I have pulled the latest code of main branch to run again and the problem still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar questions.

### Question

I have trained a yolov6lite-m model following the custom dataset tutorial provided. For evaluation, I realised that there is currently no available configuration file for the yolov6lite-m model. Hence, my question is how to do set up the configuration file for yolov6lite-m model for evaluation run? And also more importantly, how do I ensure that the images are scaled to 320 x 320 before inference? So that I can ensure that the results are as accurate as possible.

Thank you!

Thanks!

**Comment by jethrolow** [comment]
OK noted thanks a lot for the clarification!

**Comment by jethrolow** [comment]
Hi sorry to open this again.

I would just like to check, when performing training, should I set the size to 320 for yolov6lite-m training , or can I leave it at the default value of 640?

The reason I am asking is because I saw that the yolov6_lite-m config file has a scale_size = 0.5 at the model backbone. Hence, I am not sure whether this scale_size parameter will resize the images before feeding into the train loop.

Thanks!
